templates/getting_started/faq.md

```diff
@@ -493,51 +493,40 @@ model.fit(dataset, epochs=10, validation_data=val_dataset)
 
 ---
 
-### How can I regularly save Keras models during training?
+### How can I ensure my training with Keras can withstand program interruptions?
 
 To ensure the ability to recover from an interrupted training run at any time (fault tolerance),
-you should use a callback that regularly saves your model to disk. You should also set up
-your code to optionally reload that model at startup. Here's a simple example.
+you should use a `tf.keras.callbacks.experimental.BackupAndRestore` that regularly saves your training progress,
```

Comment:
Note: it sounds like we should move it out of experimental by now, no?

Reply:
Yes, we should. It's considered stable now. I'll take an action item to request the graduation from the API owners.